zanata-war/src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -78,4 +78,387 @@
+  <!--<navigation-rule>
+    <from-view-id>/account/inactive_account.xhtml</from-view-id>
+    <navigation-case>
+      <from-action>#{inactiveAccountAction.sendActivationEmail}</from-action>
+      &lt;!&ndash; TODO [CDI] end conversation &ndash;&gt;
+      &lt;!&ndash;<end-conversation />&ndash;&gt;
+      <to-view-id>/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <from-action>#{inactiveAccountAction.changeEmail}</from-action>
+      &lt;!&ndash; TODO [CDI] end conversation &ndash;&gt;
+      &lt;!&ndash;<end-conversation />&ndash;&gt;
+      <to-view-id>/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+
+  </navigation-rule-->>
+
+  <!-- Admin -->
+  <navigation-rule>
+    <from-view-id>/admin/roledetail.xhtml</from-view-id>
+    <navigation-case>
+      <from-outcome>success</from-outcome>
+      <to-view-id>/admin/rolemanager.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+  </navigation-rule>
+
+  <navigation-rule>
+    <from-view-id>/admin/userdetail.xhtml</from-view-id>
+    <navigation-case>
+      <from-outcome>success</from-outcome>
+      <to-view-id>/admin/usermanager.xhtml</to-view-id>
+    </navigation-case>
+  </navigation-rule>
+
+  <navigation-rule>
+    <from-view-id>/admin/server_configuration.xhtml</from-view-id>
+    <navigation-case>
+      <from-action>#{serverConfigurationBean.update}</from-action>
+      <from-outcome>success</from-outcome>
+      <to-view-id>/admin/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+  </navigation-rule>
+
+  <!-- Version Group -->
+  <navigation-rule>
+    <from-view-id>/version-group/create_version_group.xhtml</from-view-id>
+    <navigation-case>
+      <from-action>#{versionGroupHome.persist}</from-action>
+      <if>#{versionGroupHome.managed}</if>
+      <to-view-id>/version-group/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+  </navigation-rule>
+
+  <!-- User profile -->
+  <navigation-rule>
+    <from-view-id>/profile/create_user.xhtml</from-view-id>
+    <navigation-case>
+      <from-outcome>success</from-outcome>
+      <to-view-id>/home.xhtml</to-view-id>
+      <!-- TODO If we redirect, the global message will be lost. -->
+      <!--<redirect/>-->
+    </navigation-case>
+    <navigation-case>
+      <from-outcome>failure</from-outcome>
+      <to-view-id>/profile/create_user.xhtml</to-view-id>
+    </navigation-case>
+  </navigation-rule>
+
+  <navigation-rule>
+    <from-view-id>/profile/merge_account.xhtml</from-view-id>
+    <navigation-case>
+      <from-action>#{accountMergeAction.cancel}</from-action>
+      <from-outcome>cancel</from-outcome>
+      <to-view-id>/dashboard/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+  </navigation-rule>
+
+  <!-- Project -->
+  <navigation-rule>
+    <from-view-id>/project/project.xhtml</from-view-id>
+    <navigation-case>
+      <from-outcome>projectSlugUpdated</from-outcome>
```

Comment:
Some other places are using dashes. What about `project-slug-updated`... or conversely switch everything else to camel case. I think this is the odd one out.